### PR TITLE
Added failovermac option to config

### PR DESCRIPTION
**Pull request by killianmuldoon**
Added failOverMac option to the config for bonding cni. Introduced as a solution to issues #11  and #14

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,6 +49,7 @@ Given the following network configuration:
 		"type": "bond",
 		"mode": "active-backup",
 		"miimon": "100",
+        "failOverMac": "1",
```
Comment (zshi-redhat) [inline README.md:52]: remove double quotes for value 1, otherwise, it complains bondingConfig.failOverMac type error.

Comment (killianmuldoon) [reply README.md:52]: Good catch - thanks for that.

**Pull request merged**